### ISSUE TITLE
Version specific docsets

### ISSUE THREAD

**Issue by genotrance** [schmidh/Nim-Docset-Files#2]
It will be great if version specific docsets could also be generated and posted somewhere so that it can be possible to refer to a doc for an older version if so required.

I was posting these on the forum but wasn't quite fast enough since it was a manual process.

https://forum.nim-lang.org/t/5050

**Comment by schmidh** [comment]
Hi Ganesh,

is there really an application case for that? 

If you really need an older documentation version you could always use the original. But why have it in Dash/Zeal?

**Comment by genotrance** [comment]
> you could always use the original. But why have it in Dash/Zeal?

Dash/Zeal give such a nice experience locally, why only offer latest docs? In fact, I wonder why this isn't a feature available to users by default in the tool - the ability to list all versions and pick which one(s) user wants to view. I also noticed you removed all the older versions I had uploaded to the Nim repo which further defeats the work that was done already. There was no good reason to remove past work, now even if the tool added that functionality, (the schema already supported listing past versions), only the latest 1.0.6 you created would be available.

Anyway, point is that when Nim 1.2.x comes out, 1.0.x will still have active users and there might be those who want to use it but docs won't be easy to find. User will find out the hard way that a proc they saw in the docs that was added in 1.2 isn't really available.

Not everyone can expect to update their compiler to the latest/greatest. Especially in production / commercial endeavors.
